clang/test/SemaSYCL/sycl-dllimport-dllexport.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %clang_cc1 -triple spir64-unknown-linux-sycldevice -fsycl-is-device -fms-extensions -DTEST %s -o /dev/null 2>&1 | FileCheck %s --check-prefixes CHECK
```

Comment:
Why the triple in this run command is set to `spir64-unknown-linux-sycldevice` and all other commands `spir64-unknown-windows-sycldevice`?

Please, add a comment clarifying the intent, if it's not just a bug.

Reply:
Just negative test.  Get random combination.